casper/src/test/scala/coop/rchain/casper/genesis/contracts/PoSSpec.scala

```diff
@@ -9,5 +9,5 @@ class PoSSpec
     extends RhoSpec(
       CompiledRholangSource("PoSTest.rho"),
       Seq.empty,
-      20.seconds
+      60.seconds
```

Comment:
it looks like we should have a default timeout for the timeout parameter in RhoSpec so we only make one change in situations like this